Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.cpp

```diff
@@ -139,6 +149,54 @@ namespace AZ
             return success;
         }
 
+        void SceneProcessingConfigSystemComponent::LoadScriptSettings()
```

Comment:
This PR is about loading the configuration. The next few submissions will address how this configuration will be used inside the script behavior inside of the Scene API.

In effect this is the back end to present the script settings for other clients such as the UX or script behavior.

Reply:
Should it be hidden behind a feature flag until it's completed, then?

Reply:
This is strictly an opt-in feature, which by default is a feature toggled feature. 
This part of the solution is testable (unit tests) and delvers some of the functionality, so it can be checked in a part of a "under pinning of the future submits"